app/build.gradle.kts

```diff
@@ -328,12 +332,48 @@ tasks {
 
 afterEvaluate {
     // Disable Fdroid-specific tasks that we want to exclude
-    val tasks = tasks.withType<GoogleServicesTask>() +
+    val fdroidTasksToDisable = tasks.withType<GoogleServicesTask>() +
         tasks.withType<InjectMappingFileIdTask>() +
         tasks.withType<UploadMappingFileTask>()
-    tasks
+    fdroidTasksToDisable
         .filter { it.name.contains("Fdroid") }
         .forEach { it.enabled = false }
+
+    tasks.named("bundle") {
+        finalizedBy("renameAabFiles")
+    }
+    tasks.register("renameAabFiles") {
+        group = "build"
+        doLast {
+            val bundlesDir = "${layout.buildDirectory.get()}/outputs/bundle"
+            println("bundlesDir: $bundlesDir")
+            renameFile(
+                "$bundlesDir/standardDebug/com.x8bit.bitwarden-standard-debug.aab",
+                "com.x8bit.bitwarden.dev.aab"
```

Comment:
Do we need the whole path here?

```kotlin
            renameFile(
                "$bundlesDir/standardDebug/com.x8bit.bitwarden-standard-debug.aab",
                "$bundlesDir/standardDebug/com.x8bit.bitwarden.dev.aab"
            )
```

Reply:
Moved forward with it for now so we can fix builds (when https://github.com/bitwarden/android/pull/4458 is also merged) but we're planning on revisiting this to move builds to a different path, we can check this too while doing that!